tests/Console/Command/test-project/.php_cs

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
I don't like that I needed to add this test assets. maybe someone can give me a hint, what I could use instead?

Reply:
you can move it to `tests/Fixtures`, and then it gonna be excluded everywhere where needed